app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -32,6 +32,27 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
     private var selectedAccountUuid: String? = null
     private var selectedFolderId: Long? = null
 
+    override fun onSaveInstanceState(outState: Bundle) {
+            // Use local variable to avoid mutable property issues.
+            // putString supports null storage, but we will type check for consistency
+            var localSelectedAccountUuid = selectedAccountUuid
+            var localSelectedFolderId = selectedFolderId
+            var localSelectedFolderSummary = unreadFolder.summary.toString()
+
+            if (localSelectedAccountUuid != null) {
+                outState.putString(SELECTED_ACCOUNT_UUID_STATE, localSelectedAccountUuid)
+            }
+
+            if (localSelectedFolderId != null) {
+                outState.putLong(SELECTED_FOLDER_ID_STATE, localSelectedFolderId)
+            }
+
+            if (localSelectedFolderSummary != null) {
+                outState.putString(SELECTED_FOLDER_SUMMARY, localSelectedFolderSummary)
+            }
+        super.onSaveInstanceState(outState)
```

Comment:
We usually call through to the super method first. Please move this call to the top of the method.